consumer.go

```diff
@@ -162,8 +187,70 @@ func (c *Consumer) run() {
 			}
 
 		case <-c.done:
+			log.Println("Consumer initiating shutdown sequence")
```

Comment:
Do we want all of these printlns?  Not really sure how chatty or not this client generally is.

Reply:
So I think these are OK, just from looking at Onebox output. This path is only hit on shutdown. I personally like a bit of verbose start/stop logging it can help when debugging, however if we find it too chatty I don't mind removing some. For now though we'll probably keep it.